test/cmd/deploy_jolokia.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+source "$(dirname "${BASH_SOURCE}")/../../hack/lib/init.sh"
+trap os::test::junit::reconcile_output EXIT
+
+os::test::junit::declare_suite_start "cmd/create"
+
+os::cmd::expect_success 'make build'
+
+TEST_DIR=$(pwd)
+
+os::cmd::expect_success 'oc new-project metric'
+
+os::cmd::expect_success 'oc create -f "$TEST_DIR"/spark-metrics-template.yaml'
+
+os::cmd::expect_success 'oc new-app --template=spark -p MASTER_NAME=master -p WORKER_NAME=worker -p SPARK_METRICS_ON=jolokia'
+
+os::cmd::try_until_text 'oc get pods' 'worker'
```

Comment:
The scope of this test can be hyper-focused on just jolokia metrics deployment, since deploy.sh covers basic deployment (and will cover the signal handling in an update as discussed on slack).  The general deploy stuff can be dropped.

So, all we need here is to check that the log states jolokia metrics have been deployed, and then to be complete we should check that the endpoint can be hit on the master (there is no service created for the worker). Since the  part of launch.sh that sets up the metrics port is common to worker/master, checking the master is sufficient.  So something like this for both master and worker (cmd suite has trouble with parentheses in text due to regex stuff, so .* is a workaround)

    os::cmd::try_until_text 'oc log dc/master' 'Starting master with jolokia metrics enabled.*deprecated'

and then expose the master-jolokia service, parse out the route from "oc get route" and test the output from a curl like

    curl master-jolokia-cmd-ffabd5.127.0.0.1.nip.io/jolokia/